cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,112 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.task('generateWorker').then(generateWorker => {
+      worker = generateWorker;
+    });
+
+    cy.viewport(1920, 1080);
+    cy.visit('/');
+  });
+
+  function addNewWorker(n) {
+    for (let i = 0; i < n; i++) {
+      cy.get('.btn#addNewRecordButton').contains('Add').click();
+      cy.findByPlaceholder('First Name').type(worker.firstName);
+      cy.findByPlaceholder('Last Name').type(worker.lastName);
+      cy.findByPlaceholder('name@example.com').type(worker.email);
+      cy.findByPlaceholder('Age').type(worker.age);
+      cy.findByPlaceholder('Salary').type(worker.salary);
+      cy.findByPlaceholder('Department').type(worker.department);
+      cy.get('.btn').contains('Submit').click();
+    }
+  }
+
+  it('Rows count selection', () => {
+    cy.get('.rt-tr-group').should('have.length', 10);
+    cy.get('select').select('5');
+    cy.get('.rt-tr-group').should('have.length', 5);
+  });
+
+  it('pagination', () => {
+    cy.get('select').select('5');
+    cy.get('.-btn:contains("Next")').should('have.attr', 'disabled');
+    cy.get('.-totalPages').should('have.text', '1');
+    addNewWorker(3);
+    cy.get('.-totalPages').should('have.text', '2');
+  });
+
+  it('should provide ability to Add new worker', () => {
+    addNewWorker(1);
+  });
+
+  it('Validate data in worker row after creating worker', () => {
+    addNewWorker(1);
+    cy.get('.rt-tbody').should('contain', worker.firstName);
+    cy.get('.rt-tbody').should('contain', worker.lastName);
+    cy.get('.rt-tbody').should('contain', worker.email);
+    cy.get('.rt-tbody').should('contain', worker.age);
+    cy.get('.rt-tbody').should('contain', worker.salary);
+    cy.get('.rt-tbody').should('contain', worker.department);
+  });
+
+  it('should provide ability to Delete worker', () => {
+    cy.get('[title="Delete"]').eq(0).click();
+  });
+
+  it('should provide ability to Delete ALL workers', () => {
+    function deleteAllElements() {
+      cy.get('[title="Delete"]').should('exist');
+      let deletedCount = 0;
+
+      while (deletedCount < 3) {
+        cy.get('[title="Delete"]').first().click();
+        cy.get('[title="Delete"]').should('exist');
```

Comment:
on the last circle, this assertion fails
![image](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/ea51243e-f2b3-40d8-bd8b-f448fe1bfd27)